### ISSUE TITLE
NTR: Methylmalonylcarnitine high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8835]
New term request Methylmalonylcarnitine high in urine (elthat_100803150439): C4-DC; C4-dicarboxylic-carnitine

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000392
- label: Elevated urinary methylmalonylcarnitine level
- parents: [Abnormal concentration of acylcarnitine in the urine (HP:0500170)](https://hpo.jax.org/app/browse/term/HP:0500170)
- pmid: 
- def: The amount of methylmalonylcarnitine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8835
- synonyms:
